.travis.yml

```diff
@@ -99,7 +80,8 @@ before_install:
   export LD_LIBRARY_PATH=${CUDA_HOME}/lib64:${LD_LIBRARY_PATH} &&
   export PATH=${CUDA_HOME}/bin:${PATH}; fi
 script:
-- ./scripts/travis_build_and_test.sh
+  - docker run --rm --user='root' -v ${TRAVIS_BUILD_DIR}:/usr/src/myapp -w /usr/src/myapp trws/raja-testing:$IMG chown -R raja /usr/src/myapp
```

Comment:
I only have 1 comment - changing the dir name from `/usr/src/myapp` to something RAJA-y, just to keep the travis logs clear.